genx/setup.cfg

```diff
@@ -1,14 +1,51 @@
-[sdist]
-formats=gztar
```

Comment:
This is to use .tar.gz as default sdist, why did you delete it?

Reply:
It is the default on linux anyway [1]. On windows it defaults to zip.

[1] https://docs.python.org/3/distutils/sourcedist.html